Makefile

```diff
@@ -12,6 +12,12 @@ STORAGE_INIT_IMG ?= storage-initializer:latest
 CRD_OPTIONS ?= "crd:trivialVersions=true"
 KFSERVING_ENABLE_SELF_SIGNED_CA ?= false
 
+# CPU/Memory limits for controller-manager
+CPU_LIMIT ?= 100m
```

Comment:
Can you namespace these with KFSERVING_CONTROLLER

Reply:
I was hesitant to do so because it would result in a long variable name, and I wasn't sure if there would be other configurable CPU/memory limits in the future.

Reply:
should I still go ahead and change the variable names to include `KFSERVING_CONTROLLER`?

Reply:
Yeah please.

Reply:
done